src/common/types/core.js

```diff
@@ -112,6 +109,8 @@ type User = {
     type: 'user',
 };
 
+type User = UserMini;
```

Comment:
There were more "mini definitions" than not, so I changed this to an alias. The definitions were exactly the same.